src/libraries/Common/src/Microsoft/Win32/SafeHandles/SafeEvpPkeyCtxHandle.Unix.cs

```diff
@@ -26,5 +27,51 @@ protected override bool ReleaseHandle()
         }
 
         public override bool IsInvalid => handle == IntPtr.Zero;
+
+        internal static SafeEvpPKeyCtxHandle CreateFromEvpPkey(SafeEvpPKeyHandle evpPkey)
+        {
+            return Interop.Crypto.EvpPKeyCtxCreate(evpPkey);
+        }
+
+        internal void ConfigureForRSASign(HashAlgorithmName hashAlgorithm, RSASignaturePaddingMode padding)
+        {
+            Interop.Crypto.CryptoNative_ConfigureForRsaSign(this, padding, hashAlgorithm);
+        }
+
+        internal void ConfigureForRSAVerify(HashAlgorithmName hashAlgorithm, RSASignaturePaddingMode padding)
+        {
+            Interop.Crypto.CryptoNative_ConfigureForRsaVerify(this, padding, hashAlgorithm);
+        }
+
+        internal void ConfigureForECDSASign()
+        {
+            Interop.Crypto.EvpPKeyCtxConfigureForECDSASign(this);
+        }
+
+        internal void ConfigureForECDSAVerify()
+        {
+            Interop.Crypto.EvpPKeyCtxConfigureForECDSAVerify(this);
+        }
+
+        internal bool TryGetSufficientSignatureSizeInBytesCore(
```

Comment:
This is redundant with "get rid of these methods", but... why does this end in "Core"?

Reply:
I'll address this separately after fixing build errors because it doesn't seem super blocking